.travis.yml

```diff
@@ -18,20 +18,19 @@ matrix:
     - php: 5.5
       env: deps=low SYMFONY_DEPRECATIONS_HELPER=weak
     - php: 5.6
-      env: SYMFONY_VERSION="2.8.*@dev"
-    - php: 5.6
+      env: SYMFONY_VERSION="3.2.*"
+    - php: 7.1
       env: deps=dev
   allow_failures:
     - php: nightly
-    - env: deps=dev
 
 env:
   global:
     - deps=no
 
 before_install:
   - composer self-update
-  - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update symfony/symfony:${SYMFONY_VERSION}; fi;
+  - if [ "$SYMFONY_VERSION" != "" ]; then jq "(.require, .\"require-dev\")|=(with_entries(if .key|test(\"^symfony/\") then .value|=\"${SYMFONY_VERSION}\" else . end))" composer.json|ex -sc 'wq!composer.json' /dev/stdin; fi;
```

Comment:
Old approach replaced to be able to keep paths used in tests in `../vendor/symfony/{component}` format, instead of introducing switches in tests to determine if `../vendor/symfony/symfony/{component}` exists. Simple regex not used to keep `conflicts` section without change.

Reply:
much simpler: `composer require symfony/lts` (be careful, versions there are just 2 and 3, not 2.8 and 3.4)

Reply:
Good idea, though you already said disadvantages yourself. I would like to keep ability to pick exact symfony version